app/services/fees/calculator_test.go

```diff
@@ -0,0 +1,190 @@
+/*
```

Comment:
It would be good to check whether the mocked methods have been called or not, depending on the scenarios.